lib/cppunit_server.rb

```diff
@@ -1,4 +1,8 @@
 require 'mumukit'
 
+Mumukit.configure do |config|
+  config.docker_image = 'mumuki/mumuki-cppunit-worker'
```

Comment:
The runner is not really downloading it, but Travis script. In production, we will have to do that as part of the deploy script. 

As an additional note, the Docker image is being uploaded manually, but that could be also automated ad part of the travis build: https://docs.travis-ci.com/user/docker/#Pushing-a-Docker-Image-to-a-Registry